requirements.txt

```diff
@@ -15,6 +15,7 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+appdirs==1.4.4
```

Comment:
Maybe a silly question, but for our purposes wouldn't it be enough to just replace `appdirs.user_cache_dir` with 

```python
import sys

if  sys.platform == "a":
    return "path_a"
elif sys.platform == "b":
    return "path_b"
....
```
Or is there something more complicated going in the background?

Reply:
I guess yes... But it's only 10KB big, has no dependencies, and takes all the responsibility from us 😄 On top of that we don't need to test anything. The actual imlementation is pretty much what you wrote: https://github.com/ActiveState/appdirs/blob/8eacfa312d77aba28d483fbfb6f6fc54099622be/appdirs.py#L264

Reply:
Aaaaand [`pip` uses it](https://github.com/pypa/pip/blob/98fdc7a33140b8d4cff12b61378d65d64d1289f6/src/pip/_internal/locations/base.py#L11), for sure we're at least as cool 🤣

Reply:
Interesting:) Yeh, the testing argument is legit. Let's just outsource the logic then:)